src/renderer/components/+workloads-statefulsets/dialog/dialog.test.tsx

```diff
@@ -81,15 +88,21 @@ const dummyStatefulSet = new StatefulSet({
 
 describe("<StatefulSetScaleDialog />", () => {
   let statefulSetApi: StatefulSetApi;
+  let openStatefulSetDialog: OpenStatefulSetDialog;
+  let render: DiRender;
 
   beforeEach(() => {
-    statefulSetApi = new StatefulSetApi({
-      objectConstructor: StatefulSet,
-    });
+    const di = getDiForUnitTesting({ doGeneralOverrides: true });
+
+    di.override(createStoresAndApisInjectable, () => true);
```

Comment:
Why does this return true or boolean? What does it mean? Should be apparent by naming?